docker-compose-node-last.yml

```diff
@@ -0,0 +1,140 @@
+version: '2.1'
```

Comment:
This is the most complex docker-compose file we have. In this case, forking the file was cleaner than duplicating services within the base file.